plf-configuration/src/main/resources/conf/platform/exo-sample.properties

```diff
@@ -0,0 +1,1235 @@
+#
```

Comment:
This file contains many properties which does not apply to the community edition, they should be removed.

Reply:
Yep, we can split it into multiple files (one per addon).
* exo-ecms addon will add ecms-sample.properties
* exo-calendar addon will add calendar-sample.properties
...
like made for chat addon, WDYT ?

Reply:
It will end up with multiple properties, which is not convenient for the customers. Shoudn't we have a basic version for community version (with only properties used in community edition) and an enterprise version (the current version + properties of new addons) ?

Reply:
ok, I see, but the EE distribution will not have all addons added into it.

Reply:
done

Reply:
> ok, I see, but the EE distribution will not have all addons added into it.
yes, this is what we have today for the chat for example, so it seems acceptable.

> Done, but I 'm not really convinced by the pertinence of this change, but it was done. We 'll see how to deal with this in a dedicated project just after lightweight merge.
ok. Do you think there is a better way ? We could keep properties in each addon and let the distributions project do a concatenation of all the sample properties, but I am afraid it would complexify for little value.

Reply:
I will provide a solution after the merge, but it's better as you said to not have references to jcr & other addons in "lightweight" distribution (y)